logstash-core/lib/logstash/instrument/metric_type/counter.rb

```diff
@@ -25,14 +25,5 @@ def execute(action, value = 1)
     def value
       @counter.value
     end
-
-    def to_hash
```

Comment:
why not having this info in metric_type base? I see this as a common structure, at least for now for the types, right?